README.md

```diff
@@ -28,7 +28,7 @@ Come on in, the water's lovely. More help? Ping `jfire` or `bhousel` on
 
 ## Prerequisites
 
-* [Node.js](http://nodejs.org/) version 4 or newer
+* [Node.js](http://nodejs.org/) version 4 or newer, ([npm](https://www.npmjs.com/package/npm) version 3 or newer)
```

Comment:
node v4 comes with npm v2 by default (AFAICT) – so it could also make sense to shorten this requirement to something like `node.js version 5 or newer` (so that users don't have to worry about checking and eventually manually upgrading their npm version)?